README.md

```diff
@@ -50,6 +50,12 @@ specified with the `args` and `vals` keywords, identical to the "common-params"
 from the `global-options`. Likewise, the `role` key is optional and defaults to 'client'
 if omitted. Valid roles are "client", "server" and "all".
 
+It is also possible to enable/disable params by marking them as "enabled".
+The schema accepts either "yes" and "no" for the "enabled" keyword, as shown below:
+```
+    { "arg": "bs", "vals": [ "4k", "8k" ], "role": "client", "enabled": "no" },
+    { "arg": "rw", "vals": [ "read", "write" ], "enabled": "yes" }
+```
```

Comment:
@rafaelfolco @k-rister should this state that "enabled" is optional and if not used the behavior is enabled=yes?

Reply:
yep, I can add it.